pandas/core/internals/array_manager.py

```diff
@@ -190,7 +191,7 @@ def __repr__(self) -> str:
     def apply(
         self: T,
         f,
-        align_keys: list[str] | None = None,
+        align_keys: list[str] | None = None,  # not used for ArrayManager
```

Comment:
not worth having a separate one for AM?

Reply:
What do you mean exactly? Separate signature for ArrayManager? 
I initially remove this from the signature (since it is not used), but we have this method as an abstract method on the base class manager, and thus it gives typing errors if I remove it here.